bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php

```diff
@@ -2011,7 +2011,7 @@ protected function getCsvData(Request $request, LocaleServiceInterface $localeSe
                         if (!isset($mappedFieldnames[$field])) {
                             $mappedFieldnames[$field] = $this->mapFieldname($field, $helperDefinitions);
                         }
-                        $objectData[$field] = $fieldData;
+                        $objectData[$field] = is_array($fieldData) ? $fieldData[0] : $fieldData;
```

Comment:
what is in the rest of the array? can you give an example?

Reply:
for example an transform operator on quantity value field results in an array with one

Reply:
how is this related to this PR ?